src/Core/Geographic/Projection.js

```diff
@@ -42,7 +42,7 @@ Projection.prototype.getCoordWMTS_WGS84 = function getCoordWMTS_WGS84(tileCoord,
     if (tileMatrixSet === 'PM') {
         return this.WMTS_WGS84ToWMTS_PM(tileCoord, bbox);
     } else if (tileMatrixSet === 'WGS84G') {
-        return [tileCoord];
+        return [tileCoord.clone()];
```

Comment:
Because of https://github.com/iTowns/itowns/pull/615/files#diff-fe4e9c46a7cb968cb268ab300bafff1cR114 ? 

If it's the case, the clone must be done in the calling function, because it is the one which knows it needs a new instance.

Reply:
This function shouldn't return an object's reference
The user isn't supposed to know that are result can be changed via another object

Reply:
maybe rename in `computeCoordWMTS`

Reply:
Ok we only currently use this in cases where we need a new object, so we can leave it like that for now, it makes sense.